src/DIRAC/FrameworkSystem/Utilities/LoggingContext.py

```diff
@@ -0,0 +1,16 @@
+""" Logging context module"""
```

Comment:
All the logging code is in `DIRAC.FrameworkSystem.private.standardLogging` right? You can add this code in one of the existing module (here it feels a bit "lost").

Reply:
Indeed, but then it becomes weird to import a package coming from `<something>.private.<something_else>`.

May be the best solution would be to add it somewhere in `DIRAC.FrameworkSystem.private.standardLogging` and then call it through `DIRAC.FrameworkSystem.Client.Logger`.

Reply:
I applied the changes